### DISCUSSION TITLE
Microsoft Defender detects multiple security alerts for Selenium hub on AKS

### DISCUSSION THREAD

**Discussion by otoxolae** [SeleniumHQ/docker-selenium#2492] [Q&A]
My security team was wondering if this is intended behaviour, including this line 
/bin/sh -c /sbin/modprobe msr allow_writes=on > /dev/null 2>&1

![image](https://github.com/user-attachments/assets/9ec9adc3-ffdd-43b5-b97b-b91027980385)

**Comment by VietND96** [comment]
Can you explain a little bit on this, it means execute that command in part of container entry point?

**Reply by otoxolae** [reply]
I believe Windows Defender scans the image and found that line suspicious, here is a better image of the warning:
![image](https://github.com/user-attachments/assets/8fb410fc-f598-4511-9492-6ba330d6b526)

**Reply by otoxolae** [reply]
Do you know the purpose of this line? To them it looks like crypto-mining I guess

**Reply by diemol** [reply]
Is your Grid exposed?
You might be a victim of https://www.selenium.dev/blog/2024/protecting-unsecured-selenium-grid/

**Reply by otoxolae** [reply]
I don't think so, since this warning appeared immediately after creating the cluster. But I'll check it out, Thanks
